src/redux/userLists.ts

```diff
@@ -82,9 +160,11 @@ export const userListsClearList = listId => (dispatch, getState) => {
   });
 
   // update the list
-  const newList = { ...allNewLists[listIndex] };
-  newList.tokens = [];
-  allNewLists[listIndex] = newList;
+  if (listIndex) {
+    const newList = { ...allNewLists[listIndex] };
+    newList.tokens = [];
+    allNewLists[listIndex] = newList;
+  }
 
   dispatch({
```

Comment:
we can move this dispatch and line 173 (saveUserLists) into the if statement above

Reply:
Okay, I updated this dispatch as well as the one below to be in the `if` statements, thanks!